frontend/changelog.md

```diff
@@ -14,12 +14,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
```

Comment:
I don't want to see a commit that fix your merge conflicts in your final version of your branch.

And this modification havo no relation with your pull request...